File: src/main/java/instance/common/MemoryInit.java
package instance.common;

import cloud.common.NodeConfiguration;
import se.sics.kompics.Init;

/**
 * 
 * @author Amir Moulavi
 * @date 2011-03-18
 *
 */

public class MemoryInit extends Init {

	private NodeConfiguration nodeConfiguration;

	public MemoryInit(NodeConfiguration nodeConfiguration) {
		this.nodeConfiguration = nodeConfiguration;
	}

	public NodeConfiguration getNodeConfiguration() {
		return nodeConfiguration;
	}

}

File: src/main/java/instance/common/MemoryInit.scala
package instance.common

import se.sics.kompics.Init
import cloud.common.NodeConfiguration

/**
 * @author Amir Moulavi
 * @date 2011-07-16
 *
 */
class MemoryInit(nodeConfiguration:NodeConfiguration) extends Init {
  def getNodeConfiguration = nodeConfiguration
}